ci/jobs/projects.yaml

```diff
@@ -155,3 +155,15 @@
             - f23-np
             - rhel6-np
             - rhel7-np
+
+- project:
+    name: satellite6-pulp-automation
+    os:
```

Comment:
Any of `os`, `pulp_version` and `upgrade_pulp_version` is being used, please remove them.